tests/Auth0.ManagementApi.IntegrationTests/Auth0.ManagementApi.IntegrationTests.csproj

```diff
@@ -6,6 +6,16 @@
     <AssemblyOriginatorKeyFile>..\..\build\Auth0NetStrongName.snk</AssemblyOriginatorKeyFile>
   </PropertyGroup>
 
+  <ItemGroup>
+    <None Remove="user-import-test.json" />
```

Comment:
this is required to make the test work, right?

Reply:
Yes, that tells VS to not copy the file to the output folder, the section below tells it to embed it as a resource so that the test can read it out of the exe/dll at runtime. All this only applies to the test.